Tweeter/Scripts/app.js

```diff
@@ -9,15 +9,26 @@
         console.log(response.exists);
         if (!response.exists) {
             $("#username-ans").addClass("glyphicon-ok");
+
+            $("#RegisterButton").attr("disabled", false);
+            console.log("username did not exist in database");
+            $('#RegisterButton').show();
+
         } else {
             $("#username-ans").addClass("glyphicon-remove");
+            
+            $("input[RegisterButton]").attr("disabled", true);
+            $("input[RegisterButton]").prop("disabled", true);
```

Comment:
did you need both the .attr and .prop? Hint -- if you use 'disabled' as a class on a bootstrap <a> tag, bootstrap will take care of it for you. So you can Add or RemoveClass instead if that makes more sense to you. 

http://getbootstrap.com/css/#buttons-disabled